blockbin/src/App.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import './App.css';
+import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
+
+class App extends Component {
```

Comment:
Can we break this down into `Container`, `Header` and `DropArea`? As is it, this class feels too large already!

Reply:
lol u should see some of the components @ airbnb

Reply:
but yeah once we add functionality we should definitely refactor so might as well do it now